Header header

string[] object_name
string[] hand_name
